package.json

```diff
@@ -65,14 +65,12 @@
     "ts-loader": "^9.5.1",
     "ts-node": "^10.9.2",
     "typedoc": "^0.25.9",
-    "typescript": "^5.3.3"
+    "typescript": "^5.3.3",
+    "wscat": "^6.0.1"
   },
   "lint-staged": {
     "*.{ts,js}": [
       "eslint --fix"
     ]
-  },
-  "dependencies": {
```

Comment:
@danisharora099  why was it added in https://github.com/waku-org/js-waku/pull/2244 ?